examples/layout/lakes/step5/lib/main.dart

```diff
@@ -105,11 +105,11 @@ class MyApp extends StatelessWidget {
       children: [
         Icon(icon, color: color),
         Container(
-          margin: const EdgeInsets.only(top: 8.0),
+          margin: const EdgeInsets.only(top: 8),
```

Comment:
Hi @chalin, I think this code makes a compile error that the type is wrong. The same applies to other changes.

Reply:
Hi @mshibanami, this should not give you a compile error anymore. Which version of Flutter are you using?

Reply:
Oops, sorry. As you said, this problem was caused by the old version of flutter/dart. It's fixed after upgrading. Thanks @filiph!